src/main/steps/common/summary/utils.ts

```diff
@@ -91,6 +91,11 @@ const setkey = (userCase: Partial<CaseWithId>, key: string, language: string | u
         );
       }
       break;
+    case 'citizenUserDateOfBirthText':
+      if (userkey === 'Invalid Date' && language === 'cy') {
+        return 'Dyddiad Annilys';
```

Comment:
This text needs to come from translation infra.

Reply:
I worked on this, have changed to get translation from common.content.ts